### ISSUE TITLE
vagrant package: not all files has included

### ISSUE THREAD

**Issue by zaufi** [mitchellh/vagrant#8093]
I'm running `vagrant package` and want to add few files into a pack:
```
$ vagrant package --base Base.Windows.10 --output min-windows-10.box --include ci.builder.rsa ci.builder.ed25519 --vagrantfile Vagrantfile
==> Base.Windows.10: Exporting VM...
==> Base.Windows.10: Compressing package to: /work/vagrant/minimized-base/min-windows-10.box
==> Base.Windows.10: Packaging additional file: ci.builder.rsa
==> Base.Windows.10: Packaging additional file: Vagrantfile
```
as a result only one SSH key has added `ci.builder.rsa`, but not `ci.builder.ed25519` :(

**Comment by chrisroberts** [comment]
Hi! The list of files just needs to be comma delimited:

```
$ vagrant package --include file1,file2,...
```

I'm going to mark this to update the help output to make that more apparent. Cheers!

**Comment by kikitux** [comment]
https://github.com/mitchellh/vagrant/pull/8106

**Comment by chrisroberts** [comment]
Fixed via #8106
